test/e2e/specs/fakerp/admin.go

```diff
@@ -92,6 +92,11 @@ var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader][Fake]"
 		Expect(err).ToNot(HaveOccurred())
 	})
 
+	It("should start node-exporter correctly", func() {
```

Comment:
Is the node exporter depending on any other component in the stack in order to get deployed?

Reply:
It deploys just after alert-manager in sequence, before node-exporter.